src/stm.js

```diff
@@ -180,6 +183,7 @@ function SpeakToMe(options) {
       // create a javascript node
       scriptprocessor = audioContext.createScriptProcessor(
           bufferSize, 1, 1);
+      scriptprocessor.connect(audioContext.destination);
```

Comment:
What effect does this change have?

Reply:
For some reason this was necessary to properly connect `audiocontext` and `scriptprocessor` to make `scriptprocessor.onaudioprocess`  be fired on Chrome. Without the change the event was triggered only on FF.